### PR TITLE
Remove the from {KIE_SERVER}

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ In this command line:
...
+** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URL for {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that this {KIE_SERVER} is to join. If you are deploying several {KIE_SERVERS}, you must ensure each of the servers has a different application name.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc:54]: AB4 {CENTRAL} instance {KIE_SERVER} instance

File: doc-content/enterprise-only/openshift/freeform-monitor-deploy-assy.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@
 :what_deploying_brief: monitoring and a single {KIE_SERVER}
 
 
-To start deploying a freeform environment, deploy {CENTRAL} Monitoring and a single managed {KIE_SERVER}, which uses a PostgreSQL database server in a pod. No services are loaded on the {KIE_SERVER}. Use {LOCAL_CENTRAL} to deploy and undeploy services on the server.
+To start deploying a freeform environment, deploy {CENTRAL} Monitoring and a single managed {KIE_SERVER}, which uses a PostgreSQL database server in a pod. No services are loaded on the deployed {KIE_SERVER}. Use {LOCAL_CENTRAL} to deploy and undeploy services on the server.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/freeform-monitor-deploy-assy.adoc:10]: {KIE_SERVER} instance

File: doc-content/enterprise-only/openshift/kieserver-additional-deploy-proc.adoc [hunk 3]
```diff
@@ -31,7 +31,7 @@ In this command line:
...
-** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URL for {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that the {KIE_SERVER} is to join.
+** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URL for {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that this {KIE_SERVER} is to join.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/kieserver-additional-deploy-proc.adoc:34]: AB4

File: doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc [hunk 4]
```diff
@@ -20,7 +20,7 @@ If your environment in {OPENSHIFT} version 3 includes a {KIE_SERVER} that uses a
 oc get dc
 ----
 +
-Find the `mysql` deployment configuration that corresponds to the {KIE_SERVER}.
+Find the `mysql` deployment configuration that corresponds to the needed {KIE_SERVER}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc:23]: required {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc [hunk 5]
```diff
@@ -57,7 +57,7 @@ oc get dc/myapp-mysql -ojsonpath='{.spec.template.spec.containers[0].env[?(@.nam
 oc get svc
 ----
 +
-Find the `mysql` service that corresponds to the {KIE_SERVER}.
+Find the `mysql` service that corresponds to the needed {KIE_SERVER}.
 +
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc:61]: required {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc [hunk 6]
```diff
@@ -82,7 +82,7 @@ When prompted, enter the recorded password. The dump creation can take considera
 oc get dc
 ----
 +
-Find the `mysql` deployment configuration that corresponds to the {KIE_SERVER}.
+Find the `mysql` deployment configuration that corresponds to the needed {KIE_SERVER}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc:85]: required {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc [hunk 7]
```diff
@@ -109,7 +109,7 @@ oc get dc/myapp-mysql -ojsonpath='{.spec.template.spec.containers[0].env[?(@.nam
 oc get svc
 ----
 +
-Find the `mysql` service that corresponds to the {KIE_SERVER}.
+Find the `mysql` service that corresponds to the needed {KIE_SERVER}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc:112]: required {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc [hunk 8]
```diff
@@ -20,7 +20,7 @@ If your environment in {OPENSHIFT} version 3 includes a {KIE_SERVER} that uses a
 oc get dc
 ----
 +
-Find the `postgresql` deployment configuration that corresponds to the {KIE_SERVER}.
+Find the `postgresql` deployment configuration that corresponds to the needed {KIE_SERVER}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc:23]: required {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc [hunk 9]
```diff
@@ -62,7 +62,7 @@ oc get dc/myapp-postgresql -ojsonpath='{.spec.template.spec.containers[0].env[?(
 oc get svc
 ----
 +
-Find the `postgresql` service that corresponds to the {KIE_SERVER}.
+Find the `postgresql` service that corresponds to the needed {KIE_SERVER}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc:65]: required {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc [hunk 10]
```diff
@@ -87,7 +87,7 @@ When prompted, enter the recorded password. The dump creation can take considera
 oc get dc
 ----
 +
-Find the `postgresql` deployment configuration that corresponds to the {KIE_SERVER}.
+Find the `postgresql` deployment configuration that corresponds to the needed {KIE_SERVER}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc:90]: required {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc [hunk 11]
```diff
@@ -116,7 +116,7 @@ oc get dc/myapp-postgresql -ojsonpath='{.spec.template.spec.containers[0].env[?(
 oc get svc
 ----
 +
-Find the `postgresql` service that corresponds to the {KIE_SERVER}.
+Find the `postgresql` service that corresponds to the needed {KIE_SERVER}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc:119]: required {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 12]
```diff
@@ -39,7 +39,7 @@ To use a Git hooks directory, make the following changes:
 . If you want to customize the configuration of the Java virtual machine on the {CENTRAL} pods, select the *Enable JVM configuration* box and then enter information in any of the fields under *Enable JVM configuration*. All fields are optional. For the JVM parameters that you can configure, see <<jvm-settings-ref_{context}>>.
 . If you selected RH-SSO authentication, configure RH-SSO for {CENTRAL}:
 .. Enter the client name in the *Client name* field and the client secret in the *Client secret* field. If a client with this name does not exist, the deployment attempts to create a new client with this name and secret.
-.. If the deployment is to create a new client, enter the HTTP and HTTPS URLs that will be used for accessing the {KIE_SERVER} into the *SSO HTTP URL* and *SSO HTTPS URL* fields. This information is recorded in the client.
+.. If the deployment is to create a new client, enter the HTTP and HTTPS URLs that will be used for accessing this {KIE_SERVER} into the *SSO HTTP URL* and *SSO HTTPS URL* fields. This information is recorded in the client.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:42]: {KIE_SERVER} instance

File: doc-content/enterprise-only/openshift/template-deploy-connectcentral-proc.adoc [hunk 13]
```diff
@@ -8,10 +8,10 @@ ifeval::["{subcontext}"=="{context}-server-immutable-kjar"]
 endif::[]
 ifeval::["{subcontext}"=="{context}-freeform-server-managed"]
 :central_monitor_maven:
-To enable a connection from the {LOCAL_CENTRAL} instance that you deployed to the {KIE_SERVER},
+To enable a connection from the {LOCAL_CENTRAL} instance that you deployed to this {KIE_SERVER},
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/template-deploy-connectcentral-proc.adoc:11]: {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/template-overview-con.adoc [hunk 14]
```diff
@@ -5,22 +5,22 @@
...
-* `rhdm710-prod-immutable-kieserver.yaml` provides an immutable {KIE_SERVER}. Deployment of this template includes a source-to-image (S2I) build for one or several services that are to run on the {KIE_SERVER}. For details about this template, see <<rhdm710-prod-immutable-kieserver-ref_{context}>>.
-* `rhdm710-prod-immutable-kieserver-amq.yaml` provides an immutable {KIE_SERVER}. Deployment of this template includes a source-to-image (S2I) build for one or several services that are to run on the {KIE_SERVER}. This version of the template includes JMS integration. For details about this template, see <<rhdm710-prod-immutable-kieserver-amq-ref_{context}>>.
+* `rhdm710-kieserver.yaml` provides a {KIE_SERVER}. You can configure {KIE_SERVER} to connect to {CENTRAL}. In this way, you can set up a staging or production environment in which one {CENTRAL} manages several distinct {KIE_SERVERS}. For details about this template, see <<rhdm710-kieserver-ref_{context}>>.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/template-overview-con.adoc:23]: a {KIE_SERVER} instance
distinct {KIE_SERVER} instances

File: doc-content/enterprise-only/openshift/template-overview-con.adoc [hunk 15]
```diff
@@ -5,22 +5,22 @@
...
+* `rhdm710-prod-immutable-kieserver.yaml` provides an immutable {KIE_SERVER}. Deployment of this template includes a source-to-image (S2I) build for one or several services that are to run on this {KIE_SERVER}. For details about this template, see <<rhdm710-prod-immutable-kieserver-ref_{context}>>.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/template-overview-con.adoc:24]: this {KIE_SERVER} instance.

File: doc-content/enterprise-only/openshift/template-overview-con.adoc [hunk 16]
```diff
@@ -5,22 +5,22 @@
...
+* `rhdm710-prod-immutable-kieserver-amq.yaml` provides an immutable {KIE_SERVER}. Deployment of this template includes a source-to-image (S2I) build for one or several services that are to run on this {KIE_SERVER}. This version of the template includes JMS integration. For details about this template, see <<rhdm710-prod-immutable-kieserver-amq-ref_{context}>>.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/template-overview-con.adoc:25]: this {KIE_SERVER} instance.

File: doc-content/enterprise-only/project-data/migrating-legacy-designer-proc.adoc [hunk 17]
```diff
@@ -6,7 +6,7 @@ The legacy process designer in {CENTRAL} is deprecated in {PRODUCT} {PRODUCT_VER
 
 [NOTE]
 ====
-The {PROCESS_ENGINE} will continue to support the execution and deployment of business processes generated with the legacy designer in to the {KIE_SERVER}. If you have a legacy business process that is functioning and that you do not intend to change, it is not mandatory to migrate to the new designer at this time.
+The {PROCESS_ENGINE} will continue to support the execution and deployment of business processes generated with the legacy designer in to {KIE_SERVER}. If you have a legacy business process that is functioning and that you do not intend to change, it is not mandatory to migrate to the new designer at this time.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/migrating-legacy-designer-proc.adoc:9]: into

File: doc-content/enterprise-only/project-data/project-deploying-con.adoc [hunk 18]
```diff
@@ -3,7 +3,7 @@
 
 The deployment process can vary based on the requirements of your infrastructure.
 
-In a simple deployment of {PRODUCT}, you have one {CENTRAL} and one {KIE_SERVER}. You can use {CENTRAL} to develop your business assets and services and also to manage the {KIE_SERVER}. You can build your project in {CENTRAL} and deploy it automatically onto the {KIE_SERVER}. To enable automatic deployment, {CENTRAL} includes a built-in Maven repository. You can use {CENTRAL} to manage the {KIE_SERVER}, deploying, removing, starting, and stopping any of the services and their project versions that you have built.
+In a simple deployment of {PRODUCT}, you have one {CENTRAL} and one {KIE_SERVER}. You can use {CENTRAL} to develop your business assets and services and also to manage {KIE_SERVER}. You can build your project in {CENTRAL} and deploy it automatically onto {KIE_SERVER}. To enable automatic deployment, {CENTRAL} includes a built-in Maven repository. You can use {CENTRAL} to manage {KIE_SERVER}, deploying, removing, starting, and stopping any of the services and their project versions that you have built.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/project-deploying-con.adoc:6]: one {CENTRAL} instance and one {KIE_SERVER} instance.

File: doc-content/enterprise-only/project-data/project-deploying-con.adoc [hunk 19]
```diff
@@ -3,7 +3,7 @@
...
 
 You can also connect several {KIE_SERVERS} to the same {CENTRAL} and group them into different server configurations (in *Menu* -> *Deploy* -> *Execution Servers*). Servers belonging to the same server configuration run the same services, but you can deploy different projects or different versions of projects on different configurations. For example, you could have test servers in the `Test` configuration and production servers in a `Production` configuration. As you develop business assets and services in a project, deploy the project on the `Test` server configuration and then, when a version of the project is sufficiently tested, you can deploy it on the `Production` server configuration.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/project-deploying-con.adoc:8]: several {KIE_SERVER} instances to the same {CENTRAL} instance

File: doc-content/enterprise-only/project-data/project-deployment-central-con.adoc [hunk 20]
```diff
@@ -1,7 +1,7 @@
 [id='project-deployment-central-con_{context}']
 = Project deployment in {CENTRAL}
 
-You can use {CENTRAL} to develop your business assets and services and to manage {KIE_SERVERS} configured for project deployment. When your project is developed, you can build the project in {CENTRAL} and deploy it automatically to the {KIE_SERVER}. To enable automatic deployment, {CENTRAL} includes a built-in Maven repository. From {CENTRAL}, you can start, stop, or remove the deployment units (KIE containers) that contain the services and their project versions that you have built and deployed.
+You can use {CENTRAL} to develop your business assets and services and to manage {KIE_SERVERS} configured for project deployment. When your project is developed, you can build the project in {CENTRAL} and deploy it automatically to {KIE_SERVER}. To enable automatic deployment, {CENTRAL} includes a built-in Maven repository. From {CENTRAL}, you can start, stop, or remove the deployment units (KIE containers) that contain the services and their project versions that you have built and deployed.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/project-deployment-central-con.adoc:4]: {KIE_SERVER} instances configured

File: doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc [hunk 21]
```diff
@@ -127,7 +127,7 @@ The following example adds two version of the `Evaluation` artifact:
 ----
 ifdef::PAM[]
 +
-. Optional: if you want to be able to configure the {KIE_SERVER} to communicate with a {CENTRAL} Monitoring instance using WebSockets, add the following lines to the `pom.xml` file under the `<dependencies>` tag:
+. Optional: if you want to be able to configure the {KIE_SERVER} in the JAR file to communicate with a {CENTRAL} Monitoring instance using WebSockets, add the following lines to the `pom.xml` file under the `<dependencies>` tag:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc:130]: the {KIE_SERVER} instance in the JAR..

File: doc-content/enterprise-only/openshift/architecture-authoring-con.adoc [hunk 22]
```diff
@@ -21,9 +21,9 @@ In a single authoring environment, only one instance of {CENTRAL} is running. Mu
 
 {CENTRAL} uses persistent storage for the Maven repository and for the Git repository.
 
-A single authoring environment, by default, includes one {KIE_SERVER}. 
+A single authoring environment, by default, includes one {KIE_SERVER}.
```
[the remark this answers is not in the document]
Comment (mramendi) [reply doc-content/enterprise-only/openshift/architecture-authoring-con.adoc:24]: While I fixed this, we need to discuss the persistent use of {KIE_SERVERS} in this place and elsewhere. I am frankly not sure we need to do the deep change that *seems* to be suggested.

File: doc-content/enterprise-only/openshift/authoring-params.adoc [hunk 23]
```diff
@@ -1,6 +1,6 @@
...
-** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URLs for {CENTRAL} and {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that the {KIE_SERVER} is to join.
+** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URLs for {CENTRAL} and {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that this {KIE_SERVER} is to join.
```
[the remark this answers is not in the document]
Comment (mramendi) [reply doc-content/enterprise-only/openshift/authoring-params.adoc:3]: My mistake. This file is no longer used at all. Deleted.

File: doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc (hunk 1 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc:54]: ab4 deleted :)

File: doc-content/enterprise-only/openshift/environment-immutable-server-proc.adoc [hunk 24]
```diff
@@ -41,7 +41,7 @@ endif::PAM[]
...
+** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URL for {KIE_SERVER}. OpenShift uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. Also, the application name determines the name of the server configuration (server template) on the {CENTRAL} that this {KIE_SERVER} is to join. If you are deploying several {KIE_SERVERS}, you must ensure each of the servers has a different application name.
```
[the remark this answers is not in the document]
Comment (mramendi) [reply doc-content/enterprise-only/openshift/environment-immutable-server-proc.adoc:44]: this one got deleted too.

File: doc-content/enterprise-only/openshift/kieserver-additional-deploy-proc.adoc (hunk 3 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/kieserver-additional-deploy-proc.adoc:34]: yet another deleted file - these were from the old structure that got redone back in 7.3/7.4.

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 12 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:42]: unrelated bug fixed :)

**Pull request merged**